tests/microos/toolbox.pm

```diff
@@ -40,6 +40,16 @@ sub create_user {
     assert_script_run "echo '$user ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers.d/50-$user";
 }
 
+sub toolbox_has_repos {
+    my $output = script_output 'toolbox -r -- zypper lr -u', timeout => 180, proceed_on_failure => 1;
+    if ($output =~ m/No repositories defined/ && is_sle_micro('=6.0')) {
```

Comment:
I expect that sle-micro in dev wont have any repos, so wouldnt be better to replace the last part with `is_sle_micro && get_var(BETA)`?